packages/sources/coinapi/src/config.ts

```diff
@@ -25,3 +27,34 @@ export const makeConfig = (prefix?: string): Config => {
   }
   return config
 }
+
+export const makeWSHandler = (config?: Config): MakeWSHandler => () => {
+    const defaultConfig = config || makeConfig()
+    const getSubscription = (products: string[]) =>  ({
+      type: 'hello',
+      apikey: defaultConfig.apiKey,
+      heartbeat: false,
+      subscribe_data_type: ['exrate'],
+      subscribe_filter_asset_id: products
+    })
+    return {
+      connection: {
+        url: defaultConfig.api.baseWsURL || DEFAULT_WS_API_ENDPOINT
+      },
+      subscribe: (input) => {
+        const validator = new Validator(input, endpoint.price.customParams)
+        if (validator.error) return
+        const base = validator.overrideSymbol(NAME).toLowerCase()
+        const quote = validator.validated.data.quote.toLowerCase()
+        return getSubscription([base, quote])
+      },
+      unsubscribe: () => '',
```

Comment:
Let's rather return `undefined` here as a signal that the unsubscribe msg is undefined :)